src/Graphics/OpenGLContext/GLSL/glsl_CombinerProgramBuilder.cpp

```diff
@@ -466,7 +466,12 @@ class ShaderBlender1 : public ShaderPart
 public:
 	ShaderBlender1(const opengl::GLInfo & _glinfo)
 	{
-		if (_glinfo.isGLES2) {
+		m_glinfo = _glinfo;
+	}
+
+	void update()
+	{
+		if (m_glinfo.isGLES2) {
```

Comment:
This will update m_part each time new shader is created.
I suggest to add this check:
```
if (m_cycleType == g_cycleType)
  return;
```